### ISSUE TITLE
Create a Flatpak page

### ISSUE THREAD

**Issue by davidjharder** [getsolus/help-center-docs#460]
As we intend to only increase our flatpak support, we should have a page covering flatpak topics:

- Installing `flatpak`
- Enabling the flathub repository
- Finding flathub apps
- Installing apps
- Removing apps
- Troubleshooting
- Others?

The page should be written keeping in mind that the plan is to enable flatpak support in KDE Discover and GNOME Software

**Comment by Justinzobel** [comment]
With adding KDE's Discover (not sure about GNOME Software) I'm not sure this page will be needed. Maybe the troubleshooting section for any issues.

But in my opinion, the Plasma ISO should come with Discover and flatpak pre-installed and Flat hub enabled in user space by default. However, I'm pro Flatpak and thus biased. Perhaps a discussion on if flatpak should be pre-installed on all ISOs and if so, if Flathub should be enabled by default.

**Comment by davidjharder** [comment]
I agree the page becomes less useful if or when we follow through on our Discover/GS goals. At that point the page would be removed/changed as we write some *other* "Package Managers" page. Plus we should write docs that describe the current state of things, not what we hope they will be.

**Comment by Justinzobel** [comment]
That is true.

**Comment by TraceyC77** [comment]
Discover and GNOME Software have been rolled out, and they provide flatpak support. Any docs around this will need to go over how to manage flatpaks with them, and via command line
